metricbeat/magefile.go

```diff
@@ -90,11 +89,30 @@ func TestPackages() error {
 	return mage.TestPackages(mage.WithModulesD())
 }
 
+// Dashboards collects all the dashboards and generates index patterns.
+func Dashboards() error {
+	return mage.KibanaDashboards("module")
+}
+
+// Config generates both the short and reference configs.
+func Config() {
+	mg.Deps(metricbeat.ConfigOSS, metricbeat.GenerateDirModulesD)
+}
+
 // Update updates the generated files (aka make update).
 func Update() error {
 	return sh.Run("make", "update")
 }
 
+// NewUpdate reimplements update using magefile
+// TODO:
+// - Generate docs/fields.asciidoc
+func NewUpdate() {
```

Comment:
Why calling this NewUpdate instead of Update? not saying it's wrong, probably I'm missing some context

Reply:
Ok I saw the explanation in the description, is this the only missing piece to have feature parity?

Reply:
I'm worried this could cause confusion for a new dev, what do you think about having this commented out until we can really replace `make update`?

Reply:
Yeah, I can comment this out till everything is in place to replace `make update`.